lib/Doctrine/ORM/Query/Expr/Between.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Doctrine\ORM\Query\Expr;
+
+class Between
```

Comment:
This class should be marked as `final` since it doesn't make much sense in extending it